lib/verifiablePresentation/SSI.types.ts

```diff
@@ -23,10 +23,10 @@ export interface CredentialStatus {
 export interface Credential {
   '@context': string[];
   id: string;
-  type: string[] | string;
+  type: string[];
   credentialSubject: CredentialSubject;
   issuer: unknown;
-  issuanceDate?: string;
+  issuanceDate: string;
```

Comment:
as Niels said yesterday, this is something to look into. it might be added after a Credential signed.

Reply:
Then it should be in the VerifiableCredential

Reply:
the point is this is still an open question. the documentation states it is in `Credential` which is the same as our model, but they might be inaccurate about this

Reply:
Having it in here is fine. It means by extension if will be in VC. If the issuer is signing it could always update the value

Reply:
We can either leave it like this, another option to go about it is to have it as optional in here, and make it mandatory in the VC